polaris/polaris/management/commands/execute_outgoing_transactions.py

```diff
@@ -0,0 +1,90 @@
+import time
+from datetime import datetime, timezone
+
+from django.core.management import BaseCommand
+
+from polaris.utils import Logger
+from polaris.models import Transaction
+from polaris.integrations import registered_rails_integration as rri
+
+
+logger = Logger(__name__)
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--loop",
+            action="store_true",
+            help="Continually restart command after a specified number of seconds.",
+        )
+        parser.add_argument(
+            "--interval",
+            "-i",
+            type=int,
+            help=(
+                "The number of seconds to wait before "
+                "restarting command. Defaults to 30."
+            ),
+            default=30,
+        )
+
+    def handle(self, *args, **options):
+        if options.get("loop"):
+            while True:
+                self.execute_outgoing_transactions()
+                time.sleep(options.get("interval"))
+        else:
+            self.execute_outgoing_transactions()
+
+    @staticmethod
+    def execute_outgoing_transactions():
+        # For the time being this function is only for SEP 31 transactions
+        # Eventually we'll process transfers for SEP 6 and SEP 24 transactions
+        # here as well.
+        transactions = Transaction.objects.filter(
+            protocol=Transaction.PROTOCOL.sep31,
+            status=Transaction.STATUS.pending_receiver,
```

Comment:
is there any other reason a transaction might be pending_receiver? I don't think so but just want to make sure

Reply:
Nope, I think thats it. But thats a big assumption though. I can't think of another reason.